src/bitmap/container.rs

```diff
@@ -38,13 +39,10 @@ impl Container {
         }
     }
 
-    pub fn insert_range(&mut self, range: Range<u16>) -> u64 {
-        // If the range is larger than the array limit, skip populating the
-        // array to then have to convert it to a bitmap anyway.
-        if matches!(self.store, Store::Array(_)) && range.end - range.start > ARRAY_LIMIT as u16 {
-            self.store = self.store.to_bitmap()
-        }
```

Comment:
Did you forget to bring this part back in your PR?

Reply:
I thought that I only need to handle the inner one 😂, so I didn't handle the container. I would add another commit later.